### PR TITLE
Release Prep for 1.1.0

### PR DESCRIPTION
**Pull request by HelenCampbell**
This is a changelog and metadata update for the new 1.1.0 release of this module.

### CODE REVIEW

**Comment by HelenCampbell** [comment]
Hey @highb and @jpartlow (I wasn't sure who to ping on this, but you two seem to be the most regular contributors in the past few months). I'm doing a new release for this module and this is the release prep for it. Would you be able to review and merge for me please? Thanks!

**Comment by DavidS** [comment]
:+1:

**Comment by jpartlow** [comment]
Hi @HelenCampbell @DavidS 

Unfortunately the module work isn't quite completed.  There are three areas outstanding (https://tickets.puppetlabs.com/browse/PE-11589 is the general epic)

* Windows upgrades in PE is in final testing: https://tickets.puppetlabs.com/browse/PE-14354
* Upgrades from one AIO version to another Brandon is finishing up: https://tickets.puppetlabs.com/browse/PE-11531
* Solaris11 support is still buggy: https://tickets.puppetlabs.com/browse/PE-12299

For that later one, should we revert the solaris11 work prior to release, or simple update the documentation to make it clear that solaris11 upgrades have known issues.  Probably the later?

Josh

**Comment by HelenCampbell** [comment]
Hey @jpartlow , Following the email chain about this, I've updated the changelog and readme to include a list of known issues, would you take a look for me and ensure I'm saying the correct thing?

File: README.markdown [hunk 1]
```diff
@@ -112,6 +112,11 @@ In addition, there are several known issues with Windows:
 * On Windows Server 2003, only x86 is supported, and the `arch` parameter is ignored. If you try to force an upgrade to x64, Puppet installs the x86 version with no error message.
 * On Windows Server 2003 with Puppet Enterprise, the default download location is unreachable. You can work around this issue by specifying an alternate download URL in the `source` parameter.
  
+Specifically in the 1.1.0 Release:
+* For Windows, trigger an agent run to get Puppet to create the nessesary directory structures.
```
Comment (bmjen) [inline README.markdown]: `nessesary` => `necessary`

**Comment by bmjen** [comment]
Just some docs cleanup needed. So that we don't churn out a docs PR for signoffs. 

All occurrences of `windows` needs to be `Windows`.

**Comment by HelenCampbell** [comment]
Good catch @bmjen. Fixed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,48 @@
...
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.1.0]
+
+### Summary
+The addition of several OS support features and a considerable amount of compatibility and bug fixes. 
+
+### Known issues
+While this release adds considerable features and bug fixes the following areas are known issues and require more work:
+- For Windows, trigger an agent run to get Puppet to create the nessesary directory structures.
```
Comment (jpartlow) [inline CHANGELOG.md]: Here and below I would add 'after upgrade': 'trigger an agent run after upgrade'

Could also caveat that to `puppet resource` calls will fail until the first puppet agent run.  (You really don't need a manual puppet agent run unless you have to call puppet resource prior to the first daemonized agent run...if it's worth splitting the hair).

**Comment by highb** [comment]
:+1: from me.

**Pull request merged**